cpp/es1.3/es3_day1.cpp

```diff
@@ -0,0 +1,54 @@
+#include <iostream>
+#include <fstream>
+#include <sstream>
+#include <string>
+#include <map>
+#include <vector>
+#include <algorithm>
+
+using namespace std;
+
+bool frequenzaParole(const pair<string, int>& a, const pair<string, int>& b) {
+    return a.second > b.second;
+}
+
+int main() {
+    string nomeFile = "/home/user/Documenti/esercizi/cpp/es1.3/canto1.txt";
```

Comment:
In questo caso stai compilando il path del file all'interno dell'eseguibile. Puoi rendere il programma più indipendente se leggi questo dato dai parametri che ti passano al main. Tu in questo momento non li stai guardando, ma la firma completa di `main()` sarebbe:
```c++
int main(int argc, char **argv)
```
dove `argc` è il numero dei parametri passati all'eseguibile, mentre `argv` è un array di stringhe C che contiene gli argomenti veri e propri. Qua `nomeFile` potrebbe essere `argv[1]`.

Reply:
Ovviamente questo path può anche restare il default se NON hai parametri in ingresso...

Reply:
Questa cosa non riesco a farla, però ho provato a fare la lambda function